lib/generators/module_component/USAGE

```diff
@@ -0,0 +1,39 @@
+Description:
+    This generator will create additional files that may be needed when working with a module in vets-api
+	e.g. Controllers, Models, Serializers and Service Classes
+	
+	Important: If you try to create a component for a module that does not currently exist,
+	you will be prompted via the command line with the choice to create the module.
+	
+
+Example:
+    rails generate module_component foo controller
+
+    This will create:
+        modules/foo/app/controllers/foo/v0/foo_controller.rb
+		
+Example:
+    rails generate module_component foo model
+
+    This will create:
+        modules/foo/app/controllers/foo/v0/foo_model.rb
+		
+Example:
+    rails generate module_component foo serializer
+
+    This will create:
+        modules/foo/app/serializers/foo/v0/foo_serializer.rb
+
+Example:
+    rails generate module_component foo service
+
+    This will create:
+        modules/foo/app/serializers/foo/v0/foo_service.rb	
+		modules/foo/app/serializers/foo/v0/configuration.rb	
+
+Example: Creating multiple components at the same time
+    rails generate module_component foo controller serializer
+
+    This will create:
+        modules/foo/app/controllers/foo/v0/foo_controller.rb	
+		modules/foo/app/controllers/foo/v0/foo_serializer.rb
```

Comment:
These extra tabs seem weird. If there were three items in this list would it add even more tabs? What if there were 10, would it be tabbed so far off the page we wouldn't be able to see them?

Reply:
Oh strange, the tabbing is off. I'll fix